extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+|**cell**|`number` `[3]`|An array of 10 numbers that defines a geodesic quadrilateral cell. The first 8 elements are 4 pairs of EPSG:4979 coordinates in radians, and the last 2 elements are heights in meters above the WGS84 ellipsoid.|No|
+
+Additional properties are not allowed.
+
+#### boundingVolume.box
+
+An array of 12 numbers that define an oriented bounding box. The first three elements define the x, y, and z values for the center of the box. The next three elements (with indices 3, 4, and 5) define the x axis direction and half-length.
+The next three elements (indices 6, 7, and 8) define the y axis direction and half-length. The last three elements (indices 9, 10, and 11) define the z axis direction and half-length.
+
+* **Type**: `number` `[12]`
+* **Required**: No
+
+| Box | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![](figures/box_one.png) | ![](figures/box_quadtree_one.png) | ![](figures/box_octree_one.png)  |
+
+#### boundingVolume.region
+
+An array of six numbers that define a bounding geographic region in EPSG:4979 coordinates with the order [west, south, east, north, minimum height, maximum height]. Longitudes and latitudes are in radians, and heights are in meters above (or below) the WGS84 ellipsoid.
+
+* **Type**: `number` `[6]`
+* **Required**: No
+
+| Region | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![](figures/region.png) | ![](figures/region_quadtree.png) | ![](figures/region_octree.png)  |
+
+#### boundingVolume.cell
+
+An array of 10 numbers that defines a geodesic quadrilateral cell. The first 8 elements are 4 pairs of EPSG:4979 coordinates in degrees in counterclockwise order, and the last 2 elements are heights in meters above the WGS84 ellipsoid.
+
+* **Type**: `number` `[10]`
+* **Required**: No
+
+| Cell | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![](figures/cell.png) | ![](figures/cell_quadtree.png) | ![](figures/cell_octree.png)  |
+
+
+---
+
+### subdivision
+
+Provides information about the subdivision of the tileset.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**completeLevels**|`number`|An integer describing how many levels subdivide internally. We can assume the subdivision bitcode for all tiles on levels before this one to be `11`.|No|
+|**bufferView**|`number`|An index to the buffer view containing the subdivision buffer.|No|
+
+---
+
+### content
+
+Provides information about the content of the tileset.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**levelOffset**|`number`|An integer describing which level the content bitsream, if provided, starts at|No|
+|**levelOffsetFill**|`number`|An integer describing the state value for the content bit for all levels up to the `levelOffset`.|No|
+|**bufferView**|`number`|An index to the buffer view containing the content buffer.|No|
```

Comment:
Same comment as above. What's the behavior if `bufferView` is not defined?

Reply:
Added this to property description.